### ISSUE TITLE
Train/test split for ProSPECCTs model

### ISSUE THREAD

**Issue by robin-poelmans** [BenevolentAI/DeeplyTough#29]
Hi,

In the paper it is mentioned that when benchmarking on ProSPECCTs, the model was only trained on a subset of TOUGH-M1. 

_"To prevent information leakage, we discard all TOUGH-M1 structures with more than 30% sequence identity to any structure in any ProSPECCTs dataset as well as 20 PDB entries to which no sequence cluster could be assigned, resulting in 4862 structures and 401,366 pairs left for training."_

Can you explain in more detail how this sequence identity was calculated? (What alignment method, what identity metric,...)
Can I find a list somewhere of which structures are in your final training set for the ProSPECCTs model?

Thanks in advance.
Robin
